release/models/optical-transport/openconfig-transport-types.yang

```diff
@@ -1109,6 +1115,16 @@ module openconfig-transport-types {
     description "Ethernet compliance code: 10GBASE_SR";
   }
 
+  identity ETH_25GBASE_LR {
+    base ETHERNET_PMD_TYPE;
+    description "Ethernet compliance code: 25GBASE_LR";
+  }
```

Comment:
Please add reference, probably to `802.3cc-2017`?

Reply:
802.3 clause 114

Reply:
I added a reference to 802.3cc-2022 (CL114).